src/Json.Schema.ToDotNet.UnitTests/DataModelGeneratorTests.cs

```diff
@@ -3464,5 +3615,10 @@ public sealed class C
             string actual = generator.Generate(schema);
             actual.Should().Be(Expected);
         }
+
+        private void VerifyGeneratedFileContents(IDictionary<string, ExpectedContents> expectedContentsDictionary)
```

Comment:
>VerifyGeneratedFileContents [](start = 21, length = 27)

why did you do this? I am suspicious of helpers that consist of  a single line of code. do you project more shared validation here in future?

the downside of what you've done, taken to an extreme, is endless stepping into/out during debugging sessions. as you know, i approach unit-testing to maximize debuggability/ease of update.

Reply:
Once I added the third argument, and two of the three arguments were instance fields, I felt this was more compact.

---
In reply to: [244509301](https://github.com/Microsoft/jschema/pull/62#discussion_r244509301) [](ancestors = 244509301)